restructing the secure ff nav bar (#6780)
harness/developer-hub: docs/feature-flags/secure-ff/ff-security-compliance/roles-and-permissions.md

With Harness Feature Flags, each

:::info

> In line with the custom roles and permissions, Role Based Access Control (RBAC) is a useful tool you can set to have more control over your workflow. If you'd like to learn more about [RBAC within the Harness Platform](../../../platform/role-based-access-control/rbac-in-harness.md), do have a look at our documentation to learn more.

Reviewer: ```suggestion
In line with the custom roles and permissions, Role Based Access Control (RBAC) is a useful tool you can set to have more control over your workflow. For more information, go to [RBAC within the Harness Platform](../../../platform/role-based-access-control/rbac-in-harness.md).
```
